src/Pages/Login.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import GoogleLogin from "../library/components/GoogleLogin";
+import Logo from "../library/components/Logo";
+import CLabel from "../library/components/Label";
+import Content from "../library/components/Content";
+
+const Login = () => {
+  return (
+    <Content className="MainContainer" >
+        <Content style={{ margin: "3rem 0 2rem 0", textAlign: "center" }}>
```

Comment:
Lets avoid using inline styling. Either use StyledComponents or add a CSS class to it

Reply:
ok